frontend/src/components/Item/ItemPreviewList.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from "next/image"
+import Link from "next/link"
+import { Weight } from "../../pages/weights"
+import { routes } from "../../services/routes/routes"
+import { ProgressBar } from "../ProgressBar/ProgressBar"
+
+export type ItemPreviewProps = {
+    /** Name of item */
+    name: string
+    /** Weight */
+    weight: Weight
+    /** Slug of item */
+    slug: string
+    /** Image URL */
+    imageUrl?: string
+    /** For testing */
+    datacy?: string
+}
+
+/**
+ * Excerpt component for Item
+ * 
+ * Example:
+ * ```tsx
+ * <ItemPreviewList name="Smartphone" slug="smartphone" weight="300 g" imageUrl="https://via.placeholder.com/96.png" />
+ * ```
+ */
+export const ItemPreviewList: React.FC<ItemPreviewProps> = ({ slug, name, weight, imageUrl }) => {
+    const weightString = `${weight.isCa ? "ca." : ""}${weight.value}${weight.aditionalValue ? `- ${weight.aditionalValue}` : ""} g`
```

Comment:
Gemessen daran, dass die Weightstring generation mehrfach vorkommt. Lohnt es sich das in eine Helper Funktion auszulagern? Oder könnten sich die Strings im Laufe der Entwicklung noch in andere Richtungen verändern?